### PR TITLE
1012 process bad tx

### PR DESCRIPTION
**Pull request by rpoletaev**

### CODE REVIEW

File: packages/block/block.go [hunk 1]
```diff
@@ -136,6 +136,9 @@ func (b *Block) Play(dbTransaction *model.DbTransaction) error {
 			logger.WithFields(log.Fields{"type": consts.DBError, "error": err, "tx_hash": t.TxHash}).Error("using savepoint")
 			return err
 		}
+
+		model.IncrementTxAttemptCount(nil, t.TxHash)
```
Comment (potehinre) [inline packages/block/block.go:153]: why not within DbTransaction?

File: packages/model/transaction.go [hunk 2]
```diff
@@ -149,6 +154,10 @@ func (t *Transaction) Create() error {
 
 // IncrementTxAttemptCount increases attempt column
 func IncrementTxAttemptCount(transaction *DbTransaction, transactionHash []byte) (int64, error) {
+	defer func() {
```
Comment (potehinre) [inline packages/model/transaction.go]: i don't think that separate goroutine needed

File: packages/model/transaction.go [hunk 3]
```diff
@@ -162,3 +171,16 @@ func getTxRateByTxType(txType int8) transactionRate {
...
+func logTrBigAttemptCount(tbtx *DbTransaction, txHash []byte) {
+	t := Transaction{}
+	if err := tbtx.conn.Where("hash = ?", txHash).First(&t).Error; err != nil {
+		log.WithFields(log.Fields{"type": consts.DBError, "error": err}).Error("on getting tx by hash")
+		return
+	}
+
+	if t.Attempt >= 10 {
+		txString := fmt.Sprintf("tx_hash: %s, tx_data: %s, tx_attempt: %d", t.Hash, t.Data, t.Attempt)
+		log.WithFields(log.Fields{"type": consts.BadTxError, "tx_info": txString}).Error("logging tx attempt count")
```
Comment (potehinre) [inline packages/model/transaction.go]: tx attempts exceeded, transaction marked as bad?

**Review by potehinre — CHANGES_REQUESTED** [review]

File: packages/block/block.go (hunk 1 above)
Comment (rpoletaev) [reply packages/block/block.go:153]: separated db tx in this case allow save changes in spit of execution result

**Review by potehinre — APPROVED** [review]

**Pull request merged**